pkg/sharing/secret_exports.go

```diff
@@ -0,0 +1,98 @@
+package sharing
```

Comment:
godoc again to explain "sharing". --  Sharing is Caring?  who is sharing what with whom?